src/utils/helpers.ts

```diff
@@ -97,24 +108,112 @@ export async function getSynthData(synthId: string, tokenAddress: string) {
 }
 
 /**
- * @notice Helper function to get the YAM Synths total TVL.
- * @returns The total tvl of all yam synths.
+ * @notice Helper function to get the data for the most recent synths.
+ * @dev Can be used on the front-end to display the most recent synths.
+ * @param networkId The network / chain id of the synth deployment.
  */
-export async function getYamSynthsTotalTVL() {
+export async function getResentSynthData(networkId: number) {
+  const resentSynthData: IResentSynthsData = {};
+
+  for (const synthClassName in defaultAssetsConfig[networkId]) {
+    const synthClass = defaultAssetsConfig[networkId][synthClassName];
+    const lastSynth = synthClass.slice(-1)[0];
+    const synthId = synthClassName + "-" + lastSynth.cycle + lastSynth.year;
+    const synthData = await getSynthData(synthId, networkId);
+
+    if (synthData == undefined) {
+      break;
+    }
+
+    resentSynthData[synthClassName] = synthData;
+  }
+
+  return resentSynthData;
+}
+
+/**
+ * @notice Helper function to get the total liquidity and volume of all synths in the last 24h.
+ * @param networks Array of networks that the user wants to query.
+ */
+export async function getTotalMarketData(networks: Array<number>) {
+  let totalLiquidity = 0;
+  let totalVolume = 0;
+
+  for (const networkId of networks) {
+    for (const synthClassName in defaultAssetsConfig[networkId]) {
+      const synthClass = defaultAssetsConfig[networkId][synthClassName];
+      for (let i = 0; i < synthClass.length; i++) {
+        if (!synthClass[i].expired) {
+          const synthId =
+            synthClassName + "-" + synthClass[i].cycle + synthClass[i].year;
+          const synthData = await getSynthData(synthId, networkId);
+
+          if (synthData == undefined) {
+            break;
+          }
+
+          totalLiquidity += synthData.liquidity24h;
+          totalVolume += synthData.volume24h;
+        }
+      }
+    }
+  }
+
   const response = await axios.get(`https://api.yam.finance/tvl/degenerative`);
```

Comment:
```suggestion
const response = await axios.get<{ total: string }>(
    `https://api.yam.finance/tvl/degenerative`
  );
```

Ideally we would have an interface for the expected response, but this is the way to avoid the unsafe-member-access below.